static/js/src/advantage/subscribe/checkout/components/Taxes/Taxes.test.tsx

```diff
@@ -212,4 +218,96 @@ describe("TaxesTests", () => {
     expect(screen.getByTestId("country")).toHaveTextContent("United Kingdom");
     expect(screen.getByTestId("vat-number")).toHaveTextContent("GB123123123");
   });
+
+  it("Edit should be available for pro users", async () => {
+    global.window = Object.create(window);
+    Object.defineProperty(window, "accountId", { value: "ABCDEF" });
+
+    const initialValues = {
+      country: "GB",
+      VATNumber: "GB123123123",
+      marketPlace: "canonical-ua",
+    };
+
+    const products = [
+      {
+        product: UAProduct,
+        quantity: 1,
+      },
+    ];
+    render(
+      <QueryClientProvider client={queryClient}>
+        <Formik initialValues={initialValues} onSubmit={jest.fn()}>
+          <Elements stripe={stripePromise}>
+            <Taxes products={products} setError={jest.fn()} />
+          </Elements>
+        </Formik>
+      </QueryClientProvider>,
+    );
+
+    expect(screen.queryByRole("button", { name: "Edit" })).toBeInTheDocument();
+    userEvent.click(screen.getByRole("button", { name: "Edit" }));
+    await waitFor(() => {
+      expect(screen.getByTestId("select-country")).toBeInTheDocument();
+      expect(screen.getByTestId("field-vat-number")).toBeInTheDocument();
+      expect(screen.getByRole("button", { name: "Save" })).toBeInTheDocument();
+    });
```

Comment:
Just a nit that doesn't need to be addressed :point_down:

Reply:
It is recommended to await the `userEvent` interactions and then `waitFor` should only be used to wait for a single UI change to happen if all of the changes happen simultaneously.

```suggestion
    await userEvent.click(screen.getByRole("button", { name: "Edit" }));
    await waitFor(() => {
      expect(screen.getByTestId("select-country")).toBeInTheDocument();
    });
    
    expect(screen.getByTestId("field-vat-number")).toBeInTheDocument();
    expect(screen.getByRole("button", { name: "Save" })).toBeInTheDocument();
```

Reply:
Or even better if you simply use `await screen.findBy` because `findBy` uses `expect` wrapped in `waitFor` under the hood.

```suggestion
    await userEvent.click(screen.getByRole("button", { name: "Edit" }));
    
    const countryField = await screen.findByTestId("select-country");
    expect(countryField).toBeInTheDocument();
    expect(screen.getByTestId("field-vat-number")).toBeInTheDocument();
    expect(screen.getByRole("button", { name: "Save" })).toBeInTheDocument();
```